SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -4,23 +4,37 @@ import Mathlib.Tactic.Cases
 import Mathlib.Tactic.SplitIfs
 import Mathlib.Tactic.Tauto
 import SSA.Projects.InstCombine.LLVM.SimpSet
+import SSA.Core.Util
 
 
 namespace LLVM
 
+
+def IntN w := Option (BitVec w)
```

Comment:
nit: shouldn't it be Bitvec N (or IntW)?

Reply:
Renamed to IntW.